### PR TITLE
feat: add ips calls

### PR DESCRIPTION
**Pull request by marie-j**

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build#1037.0 ✘  
* 
  * build ✔ 
  * test ✘

**Comment by ovh-cds** [comment]
CDS Report build#1038.0 ✘  
* 
  * build ✔ 
  * test ✘

**Comment by antleblanc** [comment]
Two separate commits would be nice.

One for the `/ip` and one for `/vps`;)

File: src/vps/ips/ips.service.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+angular.module("ovh-api-services").service("OvhApiVpsIps", function ($injector) {
```
Comment (antleblanc) [inline src/vps/ips/ips.service.js]: Missing the injection from `/src/vps/vps.service.js` right?

```diff
diff --git a/src/vps/vps.service.js b/src/vps/vps.service.js
index 3f5a68a43fa2..bc6a21fb429f 100644
--- a/src/vps/vps.service.js
+++ b/src/vps/vps.service.js
@@ -7,6 +7,9 @@ angular.module("ovh-api-services").service("OvhApiVps", function ($injector) {
         },
         Aapi: function () {
             return $injector.get("OvhApiVpsAapi");
+        },
+        Ips: function () {
+            return $injector.get("OvhApiVpsIps");
         }
     };
```

**Pull request merged**